<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bakeAllAnim_window</class>
 <widget class="QWidget" name="bakeAllAnim_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>529</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bake All Anim v1.1.0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grp_frameRange">
     <property name="title">
      <string>Frame Range</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="grp_frameOptions">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nth Frame:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spn_nth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Set higher than 1 to only bake every nth frame.  eg, 1, 3, 5, 7... or 1, 5, 10, 15...</string>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_pad">
          <property name="toolTip">
           <string>If checked, make sure we place a keyframe on or after the End frame.</string>
          </property>
          <property name="text">
           <string>Pad Range?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="grp_frameRange_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spn_start">
          <property name="toolTip">
           <string>The frame to start baking from.  Note that &lt;b&gt;all animation&lt;/b&gt; on the selected tracks will be cleared, not just in this range.</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>End:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spn_end">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The frame to bake to.  Note that &lt;b&gt;all animation&lt;/b&gt; on the selected tracks will be cleared, not just in this range.</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_updateRange">
        <property name="toolTip">
         <string>Matches the animation range to the 3ds Max time slider.</string>
        </property>
        <property name="text">
         <string>Get Range from Scene</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_trackSel">
     <property name="title">
      <string>Track Selection</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="box_trackScroll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Tracks selected here will be baked.</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="box_tracks">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>211</width>
           <height>146</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grp_options">
        <property name="title">
         <string>Track Options (Fewer is faster)</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="chk_transforms">
             <property name="text">
              <string>Transforms</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="chk_modifiers">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Modifiers</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="chk_visibility">
             <property name="text">
              <string>Visibility</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="chk_materials">
             <property name="text">
              <string>Materials</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_updateTracks">
        <property name="toolTip">
         <string>Update the Track Selection list with all animated tracks in the current selection.</string>
        </property>
        <property name="text">
         <string>Get Tracks from Selection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_bakeOptions">
     <property name="title">
      <string>Bake Options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Key Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_key_type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Smooth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stepped</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_getBaked">
     <property name="toolTip">
      <string>Bake selected tracks.</string>
     </property>
     <property name="text">
      <string>Get Baked</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="bar_progress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_status">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Status Update</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
